reporting_app/static/datatable.js

```diff
@@ -277,7 +318,11 @@ var configure_dt = function(dt_config) {
                     'data': c.data,
                     'name': c.data,
                     'render': function(data, type, row, meta) {
-                        return render_data(data, type, row, meta, c.fmt)
+                        if (type == 'display') {
+                            return render_data(data, type, row, meta, c.fmt);
+                        } else {
+                            return data;
+                        }
```

Comment:
Is there any danger of this affecting anything, e.g. row highlighting?

Reply:
That should be testable relatively easily. Just change usability of a run element in your test database

Reply:
I did a manual test as recommended, and row highlighting works as normal. Turns out this would be hard to unit test, but side effects like this would be unlikely as render_data applies rules to the cell, whereas row highlighting applies rules to the row using raw data from the API query. The other thing I was worried about was the LIMS review modal, but this also uses raw API data.